src/main/scala/org/broadinstitute/dsde/rawls/model/WorkspaceModel.scala

```diff
@@ -228,6 +228,8 @@ case class WorkspacePermissionsPair(workspaceId: String,
 
 case class WorkspaceStatus(workspaceName: WorkspaceName, statuses: Map[String, String])
 
+case class BucketUsageResponse(usageInBytes: BigInt)
```

Comment:
meh, none of our other case classes are final... just don't extend them and we will be fine